walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,78 @@
 
 package com.google.sps;
 
+import java.util.*; 
 import java.util.Collection;
+import com.google.sps.TimeRange;
+import com.google.common.collect.Iterables;
 
+/** 
+ * Determines a list of available TimeRanges to schedule a meeting
```

Comment:
I believe TimeRanges should be {@link TimeRanges} or something like that.

Reply:
And same for MeetingRequest

Reply:
tbh I would omit talking about these classes and just mention what this class does at a high level. For example:

Determines a list of available time slots to schedules a meeting given ...

Reply:
fixed